tests/components/tedee/test_config_flow.py

```diff
@@ -0,0 +1,126 @@
+"""Test the Tedee config flow."""
+from unittest.mock import MagicMock
+
+from pytedee_async import TedeeClientException, TedeeLocalAuthException
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.tedee.const import CONF_LOCAL_ACCESS_TOKEN, DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+FLOW_UNIQUE_ID = "112233445566778899"
+LOCAL_ACCESS_TOKEN = "api_token"
+
+
+async def test_show_config_form(hass: HomeAssistant) -> None:
+    """Test if initial configuration form is shown."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "user"
+
+
+async def test_flow_abort(hass: HomeAssistant, mock_tedee: MagicMock) -> None:
+    """Test config flow."""
+    # initial config
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    await hass.async_block_till_done()
+    assert result["type"] == FlowResultType.FORM
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_HOST: "192.168.1.62",
+            CONF_LOCAL_ACCESS_TOKEN: "token",
+        },
+    )
+
+    assert result2["type"] == FlowResultType.CREATE_ENTRY
+    assert result2["data"] == {
+        CONF_HOST: "192.168.1.62",
+        CONF_LOCAL_ACCESS_TOKEN: "token",
+    }
```

Comment:
This can just be a mock config entry that's added, no real need to go to through the whole flow again